src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -217,6 +221,20 @@ public void TrackMetric(MetricTelemetry telemetry)
         }
 
         /// <summary>
+        /// Send a <see cref="AggregatedMetricTelemetry"/> representing aggregation of metric data.
+        /// Create a separate <see cref="AggregatedMetricTelemetry"/> instance for each call to <see cref="TrackAggregatedMetric(AggregatedMetricTelemetry)"/>.
+        /// </summary>
+        public void TrackAggregatedMetric(AggregatedMetricTelemetry telemetry)
+        {
+            if (telemetry == null)
+            {
+                telemetry = new AggregatedMetricTelemetry();
+            }
+
+            this.Track(telemetry);
+        }
+
+        /// <summary>
```

Comment:
I would love to see a flat API here:
TrackAggregatedMetric(string metricName, double value);
TrackAggregatedMetric(string metricName, double value, params string[] dimensions);